real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/GPSLocalizer.java

```diff
@@ -10,14 +10,16 @@
 import com.roboclub.robobuggy.ros.NodeChannel;
 import com.roboclub.robobuggy.ros.Publisher;
 import com.roboclub.robobuggy.ros.Subscriber;
-import com.roboclub.robobuggy.ui.LocTuple;
 
 import java.util.Date;
 
 /**
  * {@link BuggyDecoratorNode} used create {@link PoseMessage}s based upon GPS 
  * sensor input
- * 
+ *
+ * This class isn't necessarily used in real time, but more as a guideline as to what
+ * these localizers should look like.
+ *
```

Comment:
I think I prefer to leave it in just as a more accessible wiki - I would not be opposed to moving it to its own project, along with other example classes, as a sort of 'Tutorial version 2'